src/Functions/generateUUIDv7.cpp

```diff
@@ -262,27 +225,16 @@ class FunctionGenerateUUIDv7Base : public TargetSpecific::Default::FunctionGener
     ImplementationSelector<IFunction> selector;
 };
 
-template<typename FillPolicy>
-void registerUUIDv7Generator(auto & factory)
+REGISTER_FUNCTION(GenerateUUIDv7)
 {
-    static constexpr auto doc_syntax_format = "{}([expression])";
-    static constexpr auto example_format = "SELECT {}()";
-    static constexpr auto multiple_example_format = "SELECT {f}(1), {f}(2)";
-
-    FunctionDocumentation::Description description = FillPolicy::description;
-    FunctionDocumentation::Syntax syntax = fmt::format(doc_syntax_format, FillPolicy::name);
+    FunctionDocumentation::Description description = R"(Generates a UUID of version 7. The generated UUID contains the current Unix timestamp in milliseconds (48 bits), followed by version "7" (4 bits), a counter (42 bit, including a variant field "2", 2 bit) to distinguish UUIDs within a millisecond, and a random field (32 bits). For any given timestamp (unix_ts_ms), the counter starts at a random value and is incremented by 1 for each new UUID until the timestamp changes. In case the counter overflows, the timestamp field is incremented by 1 and the counter is reset to a random new start value. Function generateUUIDv7 guarantees that the counter field within a timestamp increments monotonically across all function invocations in concurrently running threads and queries.)";
```

Comment:
Minor. Can we have multiline description here? It's 700+ length description to read

Reply:
Sure ... since that applies to a related function too (`generateSnowflakeId`), I'd do this in a separate PR.